plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-fields-block/style.scss

```diff
@@ -74,3 +60,17 @@
 		}
 	}
 }
+
+.wc-block-components-address-form__address_2-toggle {
+	background: none;
+	border: none;
+	cursor: pointer;
+	font-size: inherit;
+	margin-top: $gap;
+	text-align: left;
+	width: 100%;
+
+	&::before {
+		content: "+ ";
+	}
+}
```

Comment:
I moved this part out of `.is-small, .is-medium, .is-large { }` after I noticed that the links styles were missing on mobile.

<table>
<tr>
<td valign="top">Before:
<br><br>

![store test_checkout_](https://github.com/woocommerce/woocommerce/assets/3323310/95d3cc92-4356-4923-8d4f-2835ab5acd96)

</td>
<td valign="top">After:
<br><br>

![store test_checkout_ (1)](https://github.com/woocommerce/woocommerce/assets/3323310/fc02bacc-df3a-4af3-b963-bf76308c7d0c)

</td>
</tr>
</table>

Reply:
Any reason we have `+` in CSS instead of just translation strings? seems really strange to have it isolated there.

Reply:
I removed the `+` sign from the translation based on your comment in https://github.com/woocommerce/woocommerce/pull/47160#discussion_r1596497376. I decided not to add the `+` sign into in its own `_x`, but use CSS instead, so that merchants can easily overwrite that sign if wanted.

Reply:
Ah I think I didn't communicate it correctly, instead of having the whole string there, we use the label, whatever prefix there is, can be a translation string. I initially assumed we would also remove `add`, but since we're keeping it, and it's translated, no reason to have the `+` in another place.

Reply:
No worries, @senadir. I've adjusted this part and updated the corresponding e2e tests accordingly.